spec/features/new_project_spec.rb

```diff
@@ -264,7 +264,24 @@ def app_name
     expect(gemfile).to match(/high_voltage/)
   end
 
+  it "adds and configures bourbon, neat, and refills" do
+    gemfile = read_project_file("Gemfile")
```

Comment:
In a `before(:all)`. It takes less than 10 minutes to run the generator, so we do it exactly once.